### PR TITLE
BETA: Register chiso.is-a.dev

### PR DESCRIPTION
**Pull request by raeeceip**
Added `chiso.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/chiso.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "raeeceip",
+        "email": "chiboguchisomu@gmail.com"
+    },
+    "record": {
+        "CNAME": "idk"
```
Comment (DEV-DIBSTER) [inline domains/chiso.json:7]: Invalid CNAME record.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/chiso.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "raeeceip",
+        "email": "chiboguchisomu@gmail.com"
+    },
+    "record": {
+        "CNAME": "chiso.is-a-dev"
```
Comment (DEV-DIBSTER) [inline domains/chiso.json:7]: CNAME is just loop of requested domain.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/chiso.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "raeeceip",
+        "email": "chiboguchisomu@gmail.com"
+    },
+    "record": {
+        "CNAME": "chiso.is-a.dev"
```
Comment (DEV-DIBSTER) [inline domains/chiso.json:7]: CNAME is just a loop to domain being requested.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Comment by raeeceip** [comment]
What does this mean?

**Pull request closed without merging**